README.md

```diff
@@ -4,19 +4,19 @@
 ## Content
 1. [Terraform-nomad-minio](#terraform-nomad-minio)
 2. [Compatibility](#compatibility)
-3. [Usage](#usage)
-    1. [Providers](#providers)
-    2. [Intentions](#intentions)
-4. [Requirements](#requirements)
+3. [Requirements](#requirements)
     1. [Required modules](#required-modules)
     2. [Required software](#required-software)
     3. [Other](#other)
+4. [Usage](#usage)
+    1. [Providers](#providers)
+    2. [Intentions](#intentions)
 5. [Inputs](#inputs)
 6. [Outputs](#outputs)
     1. [Example](#example)
 7. [Vault secrets](#vault-secrets)
 8. [Volumes](#volumes)
-9. [Verifyin setup](#verifying-setup)
+9. [Verifying setup](#verifying-setup)
```

Comment:
Didn't see before now, but the order should be:
```
6. [Outputs](#outputs)
7. [Vault secrets](#vault-secrets)
8. [Volumes](#volumes)
9. [Example](#example)
    1. [Verifyin setup](#verifying-setup)
```
*Remember to move the actual sections as well*